### DISCUSSION TITLE
Help with Email setup?

### DISCUSSION THREAD

**Discussion by craft211** [inventree/InvenTree#2823] [General]
I'm looking to find some help getting the email settings setup.  I have been making changes to the config file for a few days now trying our different settings and im not able to get anything work.

The System Information scree shows Email settings - Email settings not configured.

If this isn't a great place to ask this, please let me know where to go. 

Here are my current setting in config.yaml. I have tried the settings in both "Email backend configuration" and "Email" sections separately and together.

'# Email backend configuration
'# Ref: https://docs.djangoproject.com/en/dev/topics/email/
'# Available options:
'# host: Email server host address
'# port: Email port
'# username: Account username
'# password: Account password
'# prefix: Email subject prefix
'# tls: Enable TLS support
'# ssl: Enable SSL support

'# Alternatively, these options can all be set using environment variables,
'# with the INVENTREE_EMAIL_ prefix:
'# e.g. INVENTREE_EMAIL_HOST / INVENTREE_EMAIL_PORT / INVENTREE_EMAIL_USERNAME
'# Refer to the InvenTree documentation for more information

email:
  backend: 'django.core.mail.backends.smtp.EmailBackend'
  host: 'smtp.gmail.com'
  port: 587
  username: 'XXXXXXXXX@XXXXXXXX.XXX'
  password: 'XXXXXXXXXXXXXXXXX'
  sender: ''
  tls: True
  ssl: False_

Thanks for any help!

**Comment by matmair** [comment]
Hi there @craft211 !
Two things that might be blockers:
- On gmail you need to use app passwords for smtp - not your normal password - see [this link](https://support.google.com/accounts/answer/185833?hl=en)
- I can not see a sender - this also must be set (to an address your gmail-account is permitted to send from)

**Reply by craft211** [reply]
Thank you for your assistance Matmair.

I did make those corrections, but im still showing Email settings not configured. 

Is there a way to verify that the changes have taken in inventree?

Apologies for my ignorance here. Still kind of a noob with docker but im trying to learn.

Here is my settings.

email:
  backend: 'django.core.mail.backends.smtp.EmailBackend'
  host: 'smtp.gmail.com'
  port: 587
  username: 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
  password: 'Google App passwords from google account page'
  sender: 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
  tls: True
  ssl: True

Also I've tried alternating the SSL and TLS to true, as well as both true. I've also tried the different ports with Gmail. 587 and 465.

**Reply by SchrodingersGat** [reply]
@craft211 that should be sufficient. You can check here: https://github.com/inventree/InvenTree/blob/d43158ab1b8cac26d00d60a0fa012a12b04d6668/InvenTree/InvenTree/status.py#L52

Did you restart the server after making the changes to the settings configuration?
